Gruntfile.js

```diff
@@ -227,83 +193,61 @@ module.exports = function(grunt) {
                 src: 'build/test-coverage/lcov/**/*.info',
             }
         },
-        // file concatenation
-        concat: {
-            options: {
-                banner: cssFileHeader,
-            },
-            // css for the "page" template
-            pageCss: {
-                src: pageCssFiles,
-                dest: kbaseStaticDir + 'css/page_concat.css',
-                nonull: true,
-            },
-            // a single file containing all the KBase css!
-            allCss: {
-                src: allCssFiles,
-                dest: kbaseStaticDir + 'css/all_concat.css',
-                nonull: true,
-            }
-        },
 
         // check the source css files are not completely horrendous
+        // config to be updated for scss files
         stylelint: {
             options: stylelintOptions,
-            kbaseCss: {
-                src: staticFilesNoConcat,
-            },
+            concat: {
+                src: kbaseStaticDir + 'css/*.css',
+            }
```

Comment:
This will be updated in a future PR as the stylelint options that were in place were for css files.